### DISCUSSION TITLE
install HIP and CUDA support

### DISCUSSION THREAD

**Discussion by zjin-lcf** [intel/llvm#5065] [General]
Can a user install SYCL with CUDA and HIP support or have to choose just one?

Thanks

**Comment by alexbatashev** [comment]
I've previously installed both on my system and didn't have any problem with that. Do you face any particular issue?

**Reply by zjin-lcf** [reply]
Do people need to configure twice and install twice ?

**Reply by alexbatashev** [reply]
```
python ../configure.py --cuda --hip --no-werror --no-assertions --cmake-opt="-DCMAKE_INSTALL_PREFIX=/path/to/install/dir"
ninja deploy-sycl-toolchain
```

This should get you there

**Reply by zjin-lcf** [reply]
Thank you for the instructions!
